nixos/modules/services/networking/dnsmasq.md

```diff
@@ -0,0 +1,59 @@
+# Dnsmasq {#module-services-networking-dnsmasq}
```

Comment:
Can you split this in a new PR? It can be merged independently from the others.

Reply:
Sure, I've split this into #275902